helper.go

```diff
@@ -277,6 +271,69 @@ func arrayToKV(m map[string]RedisMessage, arr []RedisMessage, keys []string) map
 	return m
 }
 
+func clusterMGet(client Client, ctx context.Context, keys []string) (ret map[string]RedisMessage, err error) {
+	ret = make(map[string]RedisMessage, len(keys))
+	slotGroups := make(map[uint16][]string)
+	for _, key := range keys {
+		ks := slot(key)
+		slotGroups[ks] = append(slotGroups[ks], key)
```

Comment:
Is it possible to avoid the string slice allocation here? Maybe use a `map[uint16]int` instead?